*** PageSettingsGeneral.qml.orig	Tue Jul 30 17:08:47 2024
--- PageSettingsGeneral.qml	Thu Aug 22 20:03:11 2024
***************
*** 90,95 ****
--- 90,103 ----
  			}
  		}
  
+ 		// add for Shutdown command
+ 		MbSubMenu {
+ 			description: qsTr("Shutdown")
+ 			subpage: Component { PageSettingsShutdown {} }
+ 			property VBusItem shutdownItem: VBusItem { bind: Utils.path("com.victronenergy.shutdown", "/Shutdown") }
+ 			show: shutdownItem.valid
+ 		}
+ 
  		MbSwitch {
  			property VBusItem hasBuzzer: VBusItem {bind: "com.victronenergy.system/Buzzer/State"}
  			name: qsTr("Audible alarm")
